Engine/main/update.cpp

```diff
@@ -248,14 +248,16 @@ void update_overlay_timers()
 void update_speech_and_messages()
 {
   // we need to know if there is/was voice-over
-  const bool is_voice = channel_has_clip(SCHAN_SPEECH);
+  AudioChannelsLock lock;
+  auto *ch = lock.GetChannel(SCHAN_SPEECH);
+  const bool is_voice = ch != nullptr;
```

Comment:
`is_voice` here could be the value of `is_playing`, then we won't need the lock for the rest of the function.

Reply:
I am not sure, it does different actions depending on whether clip exists and whether it is playing below.
But I may read necessary values and unlock at once.